src/FastBroadcast.jl

```diff
@@ -222,12 +222,12 @@ end
 
 function broadcast_expr!(ex)
     ex isa Expr || return ex
-    update = findfirst(isequal(ex.head), (:(+=), :(-=), :(*=), :(/=), :(\=), :(^=), :(&=), :(|=), :(⊻=), :(÷=)))
+    update = findfirst(isequal(ex.head), (:(+=), :(-=), :(*=), :(/=), :(\=), :(^=), :(&=), :(|=), :(⊻=), :(÷=), :(.+=), :(.-=), :(.*=), :(./=), :(.\=), :(.^=), :(.&=), :(.|=), :(.⊻=), :(.÷=)))
     if update ≢ nothing
-        lhs = Expr(:call, (:(+), :(-), :(*), :(/), :(\), :(^), :(&), :(|), :(⊻), :(÷))[update], ex.args[1], ex.args[2])
+        lhs = Expr(:call, (:(+), :(-), :(*), :(/), :(\), :(^), :(&), :(|), :(⊻), :(÷))[update % 10], ex.args[1], ex.args[2])
```

Comment:
Wow, this is precarious. Could you write `update` as searching in a list of pairs of operators, so we don't do the mod 10 operation? This can easily go unaligned, and would be a hard bug to track down.

Reply:
I removed the `%`.